src/renderer/state.ts

```diff
@@ -813,8 +809,8 @@ export class AppState {
         state === InstallState.installed ||
         state == InstallState.downloaded
       ) {
-        await this.installer.remove(version);
-        if (this.installer.state(version) === InstallState.missing) {
+        await window.ElectronFiddle.removeVersion(version);
+        if (this.getVersionState(version) === InstallState.missing) {
```

Comment:
Since `window.ElectronFiddle.removeVersion` uses `ipcMain.invoke`, can we make it return the version state so we don't need another IPC round-trip?

Reply:
Good suggestion! You're right, this can be optimized.